### PR TITLE
Refined Windows package creation

### PR DESCRIPTION
**Pull request by aafemt**
Get rid of temporary files that were not deleted on script abort.
Get rid of preprocessed files in GIT-watched directories.
Clean up some obsolete stuff.
Fix several little bugs.
Style unification as requested by Paul Reeves.

### CODE REVIEW

**Comment by reevespaul** [comment]
I have no comment to make. The changes are OK.

File: builds/win32/make_all.bat [hunk 1]
```diff
@@ -62,8 +62,7 @@ for %%v in (gpre_boot build_msg codes) do (
 
 :: Firebird.conf, etc
 @copy %FB_GEN_DIR%\firebird.msg %FB_OUTPUT_DIR% > nul
-:: The line @UDF_COMMENT@ should be deleted from the target file.
-findstr /V "@UDF_COMMENT@" %FB_ROOT_PATH%\builds\install\misc\firebird.conf.in > %FB_OUTPUT_DIR%\firebird.conf
+@copy %FB_ROOT_PATH%\builds\install\misc\firebird.conf.in %FB_OUTPUT_DIR%\firebird.conf
 @copy %FB_ROOT_PATH%\builds\install\misc\databases.conf.in %FB_OUTPUT_DIR%\databases.conf >nul
```
Comment (dyemanov) [inline builds/win32/make_all.bat]: I'm just wondering, if these two files are not changed during the packaging, why they carry the .in suffix? Shouldn't they be renamed to plain .conf?

Comment (AlexPeshkoff) [reply builds/win32/make_all.bat]: Yes, should be renamed - but I suggest to do it after accepting this PR.
In order not to overload it.

**Comment by aafemt** [comment]
> I suggest to do it after accepting this PR.

Should I prepare PR for that?

**Comment by AlexPeshkoff** [comment]
If you have time & desire - definitely yes

**Pull request merged**